isthmus/src/main/java/io/substrait/isthmus/SubstraitRelVisitor.java

```diff
@@ -196,6 +197,21 @@ public Rel visit(LogicalJoin join) {
 
   @Override
   public Rel visit(LogicalCorrelate correlate) {
+    // left input of correlated-join is similar to the left input of a logical join
+    var left = apply(correlate.getLeft());
```

Comment:
remove unused vars.

Reply:
Thanks for the review. For now I have removed the unused variables. This PR is part a feature developed in iterations. The currently unused variables would be needed for building Substrait plan, planned for the upcoming PR. I have not removed the join-type variable because it helps in clarifying the expected types till the next PR is merged.